spec/README.md

```diff
@@ -107,12 +107,16 @@ defined in the default registry.
 > (such as due to the release of new CLDR versions).
 
 Updates to this specification will only reserve, define, or require
-function names or function option names
-consisting of characters in the ranges a-z, A-Z, and 0-9.
-All other names in these categories are reserved for the use of implementations or users.
+function identifiers and function option identifiers
+which satisfy either of the following two requirements:
+- Includes no namespace,
+  and has a name consisting of characters in the ranges a-z, A-Z, and 0-9.
+- Uses a namespace consisting of a single character in the range a-z.
```

Comment:
Note well: names (and thus namespaces) are CASE SENSITIVE. We should reserve both a-z and A-Z as a result.

Reply:
```suggestion
- Uses a namespace consisting of a single character in the ranges a-z and A-Z.
```